share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,128 @@
+# This file is currently being customized for quartz/toss_3_x86_64_ib.
+
+spack:
+  concretization: separately
+  view: false
+  config:
+    concretizer: clingo
+    install_tree:
+      root: /home/software/radiuss
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  packages::
+    all:
+      permissions:
+        read: world
+      providers:
+        mpi: [mvapich2]   # white238
+        #mpi: [mvapich2, openmpi]  # Is this why craylib-sci shows up?
+        # Spack defaults (since limiting the scope)
+        D: [ldc]
+        awk: [gawk]
+        blas: [openblas, amdblis]
+        daal: [intel-daal]
+        elf: [elfutils]
+        fftw-api: [fftw, amdfftw]
+        flame: [libflame, amdlibflame]
+        gl: [mesa+opengl, mesa18, opengl]
+        glx: [mesa+glx, mesa18+glx, opengl]
+        glu: [mesa-glu, openglu]
+        golang: [gcc]
+        iconv: [libiconv]
+        ipp: [intel-ipp]
+        java: [openjdk, jdk, ibm-java]
+        jpeg: [libjpeg-turbo, libjpeg]
+        lapack: [openblas, amdlibflame]
+        mariadb-client: [mariadb-c-client, mariadb]
+        mkl: [intel-mkl]
+        mpe: [mpe2]
+        #mpi: [openmpi, mpich]
+        mysql-client: [mysql, mariadb-c-client]
+        opencl: [pocl]
+        osmesa: [mesa+osmesa, mesa18+osmesa]
+        pil: [py-pillow]
+        pkgconfig: [pkgconf, pkg-config]
+        rpc: [libtirpc]
+        scalapack: [netlib-scalapack, amdscalapack]
+        sycl: [hipsycl]
+        szip: [libszip, libaec]
+        tbb: [intel-tbb]
+        unwind: [libunwind]
+        yacc: [bison, byacc]
+        uuid: [util-linux-uuid, libuuid]
+  definitions:
+    #- compilers: ['%gcc@8.3.1', '%clang@10.0.0']
+    - compilers: ['%gcc@8.3.1']
```

Comment:
The `ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01` image doesn't have `gcc@8.3.1` installed.

If you really want to build all your packages with that compiler, the two options are bootstrapping that compiler or choosing an image that has it already.  I'm pretty sure we don't want compiler bootstrapping in spack PR/develop pipelines, so that leaves only using a different image.  I'm not sure if @eugeneswalker has an ubuntu builder image with that compiler, but you can build one and push it somewhere (not DockerHub) easily enough.  I can help if you want to proceed this way.

On the other hand, if you're not tied to `gcc@8.3.1`, the easiest thing to do is switch to building your stack with `gcc@7.5.0`, similar to what we're doing with the e4s and (eventually) viz+sdk stacks.

Reply:
Thanks for letting me know about the compiler issue and process.  

That's the compiler I've been using on one of Livermore Computing's systems.  I may just switch -- until I find or create an alternative image -- since I made a number of other changes to the environment (e.g., removal of a lot of system packages as externals).